network/stream/common_test.go

```diff
@@ -94,12 +94,14 @@ func nodeBinIndexes(t *testing.T, store interface {
 type SyncSimServiceOptions struct {
 	InitialChunkCount     uint64
 	SyncOnlyWithinDepth   bool
+	Autostart             bool
 	StreamConstructorFunc func(state.Store, []byte, ...StreamProvider) node.Service
 }
 
 func newSyncSimServiceFunc(o *SyncSimServiceOptions) func(ctx *adapters.ServiceContext, bucket *sync.Map) (s node.Service, cleanup func(), err error) {
 	if o == nil {
 		o = new(SyncSimServiceOptions)
+		o.Autostart = true // start stream on by default
```

Comment:
This my result in different behaviour on Autostart default value. If `o` is nil, Autostart is true, but if `o` is not nil, but some value which does not specify Autostart, Autostart will be false like `& SyncSimServiceOptions{SyncOnlyWithinDepth: true}`, which is inconsistent. I think that the boolean should be named in respect to default value being consistent in both cases, like `NoAutosync`.

This is even visible in this PR. Options are constructed just to set Autostart to false explicitly, even if it is a default value for the field.